executor/point_get.go

```diff
@@ -113,21 +113,6 @@ func (e *PointGetExecutor) buildVirtualColumnInfo() {
 
 // Open implements the Executor interface.
 func (e *PointGetExecutor) Open(context.Context) error {
-	return nil
```

Comment:
Do you need to modify `BatchPointGetExec`?

Reply:
Done